1.preprocessing_data/README.md

```diff
@@ -2,52 +2,36 @@
 
 In this module, we present our pipeline for preprocessing the CFReT pilot data.
 
-## Illumination Correction
+## Illumination Correction (IC)
 
-To correct for illumination issues within the CFReT pilot data, we use the BaSiC method that was established in an article by [Peng et al.](https://doi.org/10.1038/ncomms14836).
-We specifically use the Python implementation of this method, called [PyBaSiC](https://github.com/peng-lab/BaSiCPy).
+To correct for illumination issues within the CFReT pilot data, we use the Background method for all plates from the [CellProfiler illumination correction modules](https://cellprofiler-manual.s3.amazonaws.com/CellProfiler-4.2.4/modules/imageprocessing.html?highlight=illumination+correction#correctilluminationapply).
 
-Illumination correction is an important step in cell image analysis pipelines as it helps with downstream processes like segmentation (more accuracy in segmenting shape and identifying objects to segment) and feature extraction (accurate measurements in intensity, texture, etc.).
+To confirm that IC was working on the images, we adjusted the brightness/contrast in CellProfiler to see if the contrast between objects and background increased and the illumination looked more even. 
 
-Being able to visualize illumination errors can be hard for some datasets.
-To test if illumination correction is necessary for this dataset, we brightened images using [Fiji](https://imagej.net/software/fiji/).
-Through this, we did observe that there was technical variation in illumination across the images (e.g vignetting), which is a common issue for Cell Painting data (Figure 1).
+## Calculate and save corrected images for all plates
 
-![Raw_to_Bright_fig](example_figs/Raw_to_Bright_fig.png)
+To perform illumination correction and save the corrected images, you will need to:
 
-> Figure 1. Image Comparison: Raw to Brightened. This figure displays the images for each channel in the same well/frame. The channel metadata (red) and Cell Painting assay stain for each channel (blue) is shown to the left of the images. The main indicator of illumination issues is seeing brighter areas, especially in the background. For example, in the `Mito` channel, you can see that on the left side, the area is brighter as compared to the top right.
+### Step 1: Open the CellProfiler GUI
 
-After using PyBaSiC (or BaSiCPy) to correct for illumination, it is hard to tell if there was any change made to the images just by looking at them unedited (Figure 2).
```

Comment:
Consider providing a short amount of text here before the shell commands referenced below to help describe what's happening in natural language.

Reply:
@d33bs I am not sure what you are referencing here. The text I am seeing is for text I removed talking about PyBaSiC. Can you clarify?

Reply:
Clarified! Thank you!